pkg/config/remote_config.go

```diff
@@ -0,0 +1,100 @@
+package config
```

Comment:
IMO we should unexport everything in this file since we don't intend for it to be used by other packages (yet)